Makefile

```diff
@@ -14,19 +14,31 @@
 
 SHELL=bash
 
-REG_PATH = $(shell bender path register_interface)
+REG_PATH = ../register_interface
 REG_TOOL = $(REG_PATH)/vendor/lowrisc_opentitan/util/regtool.py
-HJSON = rtl/cTCLS_unit/cTCLS_unit.hjson
+HJSON = rtl/ecc_wrap/ecc_sram_wrapper.hjson
 
-TARGET_DIR = rtl/cTCLS_unit
+TARGET_DIR = rtl/ecc_wrap
 
-.PHONY: gen_cTCLS
+# .PHONY: gen_cTCLS
```

Comment:
Please create new variables (`HJSON`, `TARGET_DIR`) for the different generated registers. Renaming the paths breaks other implementations.
Also, please leave the bender method for fetching the register interface path in place. While not the perfect implementation, it is better than relying on a user to have the register tool cloned in a neighboring directory.